test/lib/install.js

```diff
@@ -139,6 +139,14 @@ t.test('completion to folder', async t => {
         else
           return ['package.json']
       },
+      promises: {
+        readdir: (path) => {
```

Comment:
The environment without `fs.promises` is already be tested here: https://github.com/npm/cli/blob/73633e566661f38d78a61faa1ab43919dc5696e6/test/lib/utils/reify-finish.js#L80-L86